### ISSUE TITLE
Samples Not Loading or Playing

### ISSUE THREAD

**Issue by frogjacket** [schismtracker/schismtracker#550]
Hi. I just installed schismtracker on mint linux. When I go to the sample library, highlight a sample (AIFF, supported format), and hit enter as instructed, nothing happens. I don't know how to add samples or create instruments. Please help me. 

It says "no sample" under "quality" when highlighting an AIFF file. When I highlight a WAV file, it recognizes the quality as 16 bit, but it still won't load when I hit enter.

![image](https://github.com/user-attachments/assets/d24a798e-fa57-4f41-8426-03e2b34cd76a)

**Comment by mrpapersonic** [comment]
That build is quite old, you should consider downloading the latest binary release from https://schismtracker.org/ or building it from source. If you can replicate on the newest release please don't hesitate to reopen this issue.

**Comment by frogjacket** [comment]
Hi. I installed the latest version from source and I'm still having this issue. Samples won't load, AIFF or WAV.
![image](https://github.com/user-attachments/assets/90bb6c2a-757c-4429-af8b-fc31d6067028)

**Comment by mrpapersonic** [comment]
Can you send the example file?

**Comment by sagamusix** [comment]
The problem on the second screenshot isn't that the file isn't loading (there are valid length and speed values being displayed) but rather that this is the Sample Library (Ctrl+F3) screen and not the Load Sample screen, isn't it? AFAIK you can only preview samples from the Sample Library; in order to load them, you first need to go to the sample list (F3), select a slot you want to load the sample into and then press Enter to go to the Load Sample screen.

**Comment by mrpapersonic** [comment]
Wow I can't believe I didn't notice that...

**Comment by frogjacket** [comment]
So I loaded the samples as directed, and now I have two problems:
1. AIFF files just load as noise.
2. The application isn't producing sound. When I rendered my samples as WAV and loaded those, they played just fine. When I restarted Schismtracker it suddenly stopped giving me any audio. I tried switching the audio driver between ALSA and pulseaudio and I get nothing.

^_^;

Edit: never mind. The samples are working again. I don't know what's going on with my computer, sorry.

![image](https://github.com/user-attachments/assets/601a550e-ec9c-47f2-99cb-a2d9c1373484)
